dotcom-rendering/src/web/components/LiveBlogEpic.importable.tsx

```diff
@@ -241,6 +242,20 @@ export const LiveBlogEpic = ({
 }: Props) => {
 	log('dotcom', 'LiveBlogEpic started');
 
+	const [removeEpic, setRemoveEpic] = useState(false);
+
+	const ABTestAPI = useAB()?.api;
+	const userIsInRemoveLiveblogEpicTest = ABTestAPI?.isUserInVariant(
+		'RemoveBusinessLiveblogEpics',
+		'variant',
+	);
+
+	useEffect(() => {
+		setRemoveEpic(
+			!!userIsInRemoveLiveblogEpicTest && section == 'business',
+		);
+	}, [userIsInRemoveLiveblogEpicTest, section]);
```

Comment:
I _think_ (but please verify :) you might not need this useEffect as this is a client side only island and `useAB` will force a re-render when it sets `ABTestAPI`. If that's the case then you should be able to set `removeEpic` as a normal const rather than setting it via a useEffect.

```suggestion
  const removeEpic = (userIsInRemoveLiveblogEpicTest ?? false) && section == 'business';
```

Reply:
Yep you're right! I've made the change and tested on CODE to check and all looks good 🙂